frontend/src/features/map_planning/components/NextcloudKonvaImage.tsx

```diff
@@ -1,13 +1,12 @@
 import defaultImageUrl from '@/assets/plant.svg';
```

Comment:
I think while we are at it, we should take a look here and see if we can refactor this a little more.
`React.memo` looks promising to prevent unnecessary rerenders.
Also the image creation in the render step below seems out of place :)